ScoopInstaller/Main
# Add speedtest

Adds the speedtest CLI from Ookla, (Reference: <https://www.speedtest.net/apps/cli>)

<details close>
<summary>Description</summary>
<h2>Internet connection measurement for developers</h2>

Speedtest CLI brings the trusted technology and global server network behind Speedtest to the command line. Built for software developers, system administrators and computer enthusiasts alike, Speedtest CLI is the first official Linux-native Speedtest application backed by Ookla®.

With Speedtest CLI, you can easily:

* Measure internet connection performance metrics like download, upload, latency and packet loss natively without relying on a web browser
* Test the internet connection of your Linux desktop, a remote server or even lower-powered devices such as the Raspberry Pi® with the Speedtest Server Network™
* Set up automated scripts to collect connection performance data, including trends over time
* Use Speedtest in your programs by wrapping it in the programming language of your choice
* View test results via CSV, JSONL or JSON
</details>

Regarding <https://github.com/lukesampson/scoop/wiki/Criteria-for-including-apps-in-the-main-bucket>

- a reasonably well-known and widely used developer tool
  To be honest *the cli* is not a widely used *developer tool*, but the ookla speedtest program (gui, web etc.) is pretty well known and widely used
- the latest stable version of the program
  This is the version gotten from the official website
- the full version i.e. not a trial version
  The main purpose of this is to check your internet speed and this does that perfectly without any limitations.
- a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)
  Yeah this is fairly standard
- a non-GUI tool
  The CLI is a non-GUI tool, but other GUI based and web based tools exist and can be found at <https://www.speedtest.net/apps>
